CONTRIBUTING.md

```diff
@@ -49,16 +49,20 @@ The Git LFS teams mark issues and pull requests with the following labels:
 * `roadmap` - A checklist issue with tasks to fulfill something from the
 [roadmap](./ROADMAP.md)
 
+## Branching strategy
+
+In general, contributors should develop on branches based off of `master` and pull requests should be to `master`.
+
```

Comment:
This is :sparkles:! What do you think about adding another small note in this `##` section about our strategy for release branches, i.e., `release-1.2`, `release-1.3` and so on?